fetch_textures.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
-wget "https://drive.google.com/uc?authuser=0&id=1RoiJ2LrtO6-LXAXcqhwU0Ru_eDww9oWN&export=download" -O "minecraft_texture_atlas.png"
-wget "https://drive.google.com/uc?authuser=0&id=1F6qRQiCJiaLxazc79_odZO4AnIy1Dcta&export=download" -O "specular_texture_atlas.png"
-wget "https://drive.google.com/uc?authuser=0&id=1xDxsFRYplDht0EXmxQ3KC3tBBSB3Y6vH&export=download" -O "break_texture.png"
+wget "https://drive.google.com/uc?authuser=0&id=1RoiJ2LrtO6-LXAXcqhwU0Ru_eDww9oWN&export=download" -O "res/minecraft_texture_atlas.png" 
+wget "https://drive.google.com/uc?authuser=0&id=1F6qRQiCJiaLxazc79_odZO4AnIy1Dcta&export=download" -O "res/specular_texture_atlas.png" 
+wget "https://drive.google.com/uc?authuser=0&id=1xDxsFRYplDht0EXmxQ3KC3tBBSB3Y6vH&export=download" -O "res/break_texture.png" 
```

Comment:
Please remove the empty space at the end of each line.
```suggestion
wget "https://drive.google.com/uc?authuser=0&id=1RoiJ2LrtO6-LXAXcqhwU0Ru_eDww9oWN&export=download" -O "res/minecraft_texture_atlas.png"
wget "https://drive.google.com/uc?authuser=0&id=1F6qRQiCJiaLxazc79_odZO4AnIy1Dcta&export=download" -O "res/specular_texture_atlas.png"
wget "https://drive.google.com/uc?authuser=0&id=1xDxsFRYplDht0EXmxQ3KC3tBBSB3Y6vH&export=download" -O "res/break_texture.png"
```